resources/views/projects/my-projects.blade.php

```diff
@@ -17,55 +17,66 @@
             $user->organization->isParticipant())) ||
         ($user->context === 'individual' &&
             ($user->individual->isConsultant() || $user->individual->isConnector()) &&
-            $user->individual->isParticipant()))
-        <nav class="nav--tabbed" aria-labelledby="projects">
-            <div class="center center:wide">
-                <ul class="-mt-4 flex gap-6" role="list">
-                    @if ($user->context === 'organization')
-                        @if ($user->organization->isConsultant() || $user->organization->isConnector())
+            ($user->individual->isParticipant() || $user->individual->inProgressParticipatingProjects()->count())))
+        @if ($user->context === 'organization' ||
+            ($user->context === 'individual' && count($user->individual?->roles ?? []) > 1) ||
```

Comment:
These context comparisons should compare against a value sourced from the UserContext enum